### ISSUE TITLE
Missing mkmf-lite gem at runtime

### ISSUE THREAD

**Issue by Lqp1** [djberg96/sys-filesystem#74]
Hello,

It seems that since https://github.com/djberg96/sys-filesystem/pull/73 we new use `mkmf-lite` at runtime, but the gem is pulled as dependency only as development requirement.

The gem now fails to run properly during at runtime because if we use it in environment where mkmf-lite it's not installed by another dependency.

```
Failure/Error: require 'sys/filesystem'

LoadError:
  cannot load such file -- mkmf-lite
```

**Comment by djberg96** [comment]
My bad. I bumped the version, but forgot to make it a runtime dep. I saw the PR, but it needs a rebase to the latest main.

**Comment by djberg96** [comment]
Fixed in 1.5.2. Sorry about that.Thanks for the report.

**Comment by Lqp1** [comment]
Thanks for your reactivity :)
